apps/omg/test/omg/ethereum_client_monitor_test.exs

```diff
@@ -55,13 +66,13 @@ defmodule OMG.EthereumClientMonitorTest do
     WebSockexServerMock.shutdown(server_ref)
     true = is_pid(Process.whereis(EthereumClientMonitor))
 
+    node = Node.self()
+
     :ok =
-      pull_client_alarm(400, [
-        %{
-          details: %{node: Node.self(), reporter: EthereumClientMonitor},
-          id: :ethereum_client_connection
-        }
-      ])
+      call_while(
```

Comment:
why exactly was this rewritten?

Reply:
because the previous implementation doesn't work :D
it's best seen here: https://github.com/omisego/elixir-omg/blob/master/apps/omg_child_chain_rpc/test/omg_child_chain_rpc/web/plugs/health_test.exs#L64-L72

pin value causes that if two objects are equal then it will be a match.
if `TestHelper.rpc_call(:post, "/block.get", %{})` return 
```
%{ 
 "data" => %{"code" => "operation:service_unavailable"},
 "error" ...
}
``` 
then in function pull_client_alarm never match
ps. function call with `%{ "data" => %{"code" => "operation:service_unavailable"}}`

Reply:
Why would the endpoint return an `error`?

Reply:
this return rpc_call
```elixir
%{
  "data" => %{
    "code" => "operation:service_unavailable",
    "description" => "The server is not ready to handle the request.",
    "object" => "error"
  },
  "success" => false,
  "version" => "0.2"
}
```
even if return 
```elixir
%{
  "data" => %{
    "code" => "operation:service_unavailable",
  },
  "success" => false,
  "version" => "0.2"
}
```
will not match

Reply:
Yes you're right! The endpoints are returning a full error description and status. Fixed here: https://github.com/omisego/elixir-omg/pull/861

Reply:
having fixed in #861 this is not needed and can be reverted.